### PR TITLE
Considering nginx sites-enabled

### PR DESCRIPTION
**Pull request by lat0z**
Added the consideration of the sites-enabled default file to avoid further errors

### CODE REVIEW

**Comment by punkrokk** [comment]
Is this true on the most recent version of nginx? E.g. what we use with CentOS 8?

**Comment by lat0z** [comment]
On ubuntu 18.04 and 20.04 checking with nginx 1.14 and 1.17 there is the sites-enabled/default file, on centos 8 nginx 1.14 and 1.19 there is not

File: docs/source/install/u18.rst [hunk 1]
```diff
@@ -130,6 +130,8 @@ SSL termination, and reverse-proxy st2auth and st2api API endpoints. To set it u
 
   # Remove default site, if present
   sudo rm /etc/nginx/conf.d/default.conf
+  # Check for a default site on sites-enabled to avoid a duplicate default server error 
+  sudo rm /etc/nginx/sites-enabled/default
```
Comment (LindsayHill) [inline docs/source/install/u18.rst:134]: Could use `sudo rm -f`, so it doesn't complain if the file doesn't exist?

**Comment by lat0z** [comment]
I fixed the rm command following your recommendation @LindsayHill

**Comment by LindsayHill** [comment]
Thanks @lat0z

**Pull request merged**